php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2166,6 +2186,25 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         return $dob;
     }
 
+    /**
+     * Get the candidates DOD.
+     *
+     * @return ?string The candidate's DOD.
+     */
+    function getDoD(): ?string
```

Comment:
why is this a string and not a datetime?

Reply:
two reasons:
- The helper is used mainly in `getCandidateAge` function which calls `Utility::calculateAge`. This utility expects both date parameters to be strings.
- The similar helper `getDOB` for dates of birth also returns string.

Reply:
I agree that it should be DateTime ideally (and mentioned this in an earlier review) but given that this PR was opened in July I'm in favour with leaving it for a future refactoring effort.